.travis.yml

```diff
@@ -25,15 +25,15 @@ install:
 
 before_script:
   # Build C/C++ library and apps.
-  - mkdir -p build && cd build && cmake .. && make && cd ..
+  - mkdir -p build && cd build && CXXFLAGS="-Werror" cmake -DCMAKE_VERBOSE_MAKEFILE=ON .. && make && cd ..
```

Comment:
Maybe we can set -Werror as flag in Cmake directly, so it always used, even during development?

Reply:
https://flameeyes.blog/2009/02/25/future-proof-your-code-dont-use-werror/

You should **never** enable `-Werror` within CMake. Here's an example. What if Clang 9 adds a new linter warning we don't know of yet? Then edlib cannot be built with Clang 9, requiring patching. `-Werror` is best used in CI (possibly with multiple GCC and Clang versions) instead of pushing potential future breakage down to users.

Reply:
OK, makes sense -> we want to have it during development, but not during end users building the source code.